### PR TITLE
Edited two words in the contributing.md file

### PR DESCRIPTION
**Pull request by scarecrow21**
I've changed two words in the contributing.md file.

### CODE REVIEW

**Comment by andy5995** [comment]
That's a good change! LGTM! (I can't approve or merge)

**Comment by andy5995** [comment]
@SunyataZero I think @scarecrow21 is going to update this PR so please don't merge it yet (if you are around right now, lol)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,23 +32,27 @@ No responsibility for any financial or other damage.
 
 ## What can I do right now?
 
-* Doing code reviews
-* Writing automated tests to increase code coverage
+* Do code reviews
+* Write automated tests to increase code coverage
```
Comment (andy5995) [inline CONTRIBUTING.md]: Here, @SunyataZero , he corrected the verb tenses to match the heading `What can I do right now?`

Actually, `Do` doesn't need to be there. "What can I do?" answer... "Code reviews".. The "do" is implied.

Comment (scarecrow21) [reply CONTRIBUTING.md]: Should I change that @andy5995 ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,23 +32,27 @@ No responsibility for any financial or other damage.
...
 ## Pull Requests
 
 1. Fork the repo (if you haven't already done so)
-2. Clone it to your computer
-3. When you're ready to work on an issue, be sure you're on the **master** branch. From there, [create a separate branch](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches) (e.g. issue_32)
-4. Make your changes. If you're unsure of some details while you're making edits, you can discuss them on the ticket or in the [Gitter chat room](https://gitter.im/mindfulness-at-the-computer/Lobby).
-5. Commit your changes
-6. Push the working branch (e.g. issue_32) to your remote fork.
+2. Clone it to your computer. You can do it using `git clone url_name`.
+3. When you're ready to work on an issue, be sure you're on the **master** branch(Check using `git branch`). From there, [create a separate branch]. You can do this using `git checkout -b branch_name` where branch_name (e.g. issue_32) is the name of a new branch that you create.
```
Comment (andy5995) [inline CONTRIBUTING.md]: These brackets [ ] can be deleted.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,23 +32,27 @@ No responsibility for any financial or other damage.
...
+(https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches)
```
Comment (andy5995) [inline CONTRIBUTING.md]: Now that you've given the command-line instructions, this URL can be deleted

**Comment by andy5995** [comment]
@SunyataZero , it looks like @scarecrow21 finished fixing the changes. If you're satisfied, this is ready for merging.

**Pull request merged**